bouncer/shared/evm_vault_swap.ts

```diff
@@ -75,7 +75,7 @@ export async function executeEvmVaultSwap(
   } as const;
   const txOptions = {
     // This is run with fresh addresses to prevent nonce issues. Will be 1 for ERC20s.
-    gasLimit: srcChain === Chains.Arbitrum ? 10000000n : 200000n,
+    gasLimit: srcChain === Chains.Arbitrum ? 32000000n : 5000000n,
```

Comment:
Arbitrum's "soft" gas limit is 32M. Increased Eth's gas limit because larger messages need more gas.